dynamic/main.go

```diff
@@ -94,6 +95,34 @@ func initialSetup() (info.Provider, pfbridge.ProviderMetadata, func() error) {
 				info.SchemaPostProcessor(&packageSchema.PackageSpec)
 			}
 
+			if fullDocs {
+				// Create a custom generator for registry docs (_index.md).
+				root := afero.NewMemMapFs()
```

Comment:
AFAIK, we are just doing this in memory if `indexDocsOutDir` is `""`? We will never save it to disk. Why would we bother doing this at all?

Reply:
AIUI, `NewMemMapFs` _is_ the in-memory option.